client/shipping-label/views/purchase/steps/packages/index.js

```diff
@@ -30,25 +31,58 @@ const PackagesStep = ( {
 	const totalWeight = packageIds.reduce( ( result, pId ) => ( result + selected[ pId ].weight ), 0 );
 	const isValidWeight = packageIds.reduce( ( result, pId ) => ( result && 0 < selected[ pId ].weight ), true );
 	const isValidPackages = 0 < packageIds.length;
+	const hasAnyPackagesConfigured = all && Object.keys( all ).length;
 
-	const renderSummary = () => {
+	const getContainerState = () => {
 		if ( ! isValidPackages ) {
-			return __( 'No packages selected' );
+			return {
+				isError: true,
+				summary: __( 'No packages selected' ),
+			};
 		}
 
 		if ( ! isValidWeight ) {
-			return __( 'Weight not entered' );
+			return {
+				isError: true,
+				summary: __( 'Weight not entered' ),
+			};
 		}
 
+		let summary = '';
+
 		if ( 1 === packageIds.length && 1 === itemsCount ) {
-			return sprintf( __( '1 item in 1 package: %f %s total' ), totalWeight, storeOptions.weight_unit );
+			summary = sprintf( __( '1 item in 1 package: %f %s total' ), totalWeight, storeOptions.weight_unit );
+		} else if ( 1 === packageIds.length ) {
+			summary = sprintf( __( '%d items in 1 package: %f %s total' ), itemsCount, totalWeight, storeOptions.weight_unit );
+		} else {
+			summary = sprintf( __( '%d items in %d packages: %f %s total' ), itemsCount, packageIds.length, totalWeight, storeOptions.weight_unit );
+		}
+
+		if ( ! hasAnyPackagesConfigured ) {
+			return { isWarning: true, summary };
 		}
 
-		if ( 1 === packageIds.length ) {
-			return sprintf( __( '%d items in 1 package: %f %s total' ), itemsCount, totalWeight, storeOptions.weight_unit );
+		return { isSuccess: true, summary };
+	};
+
+	const renderWarning = () => {
+		if ( ! hasAnyPackagesConfigured ) {
+			const packagesMsg = sprintf(
+				__( 'There are no packages configured. The items have been packed individually. You can add or enable packages using the <a href="%(url)s">Packaging Manager</a>.' ),
```

Comment:
I wonder if all these URLs that take you out of the printing flow should have `target="_blank"`. @kellychoffman I'll trust you blindly on this one, I just wanted to bring it to your attention :)